extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -12,7 +12,15 @@
  */
 package org.eclipse.smarthome.binding.ntp.handler;
 
-import static org.eclipse.smarthome.binding.ntp.NtpBindingConstants.*;
+import static org.eclipse.smarthome.binding.ntp.NtpBindingConstants.CHANNEL_DATE_TIME;

```

Comment:
I don't know if we every had added a "rule" for static imports.
Other projects states e.g. as soon as the static imports is greater or equal then 3 (or 4 or 5) a wildcard should be used.
@kaikreuzer What is our guideline?

Reply:
We have an implicit rule: our code-formatter 😄 

![image](https://user-images.githubusercontent.com/2592443/34154732-d9877900-e4b6-11e7-82a4-a3b32c749353.png)

Reply:
I will revert this then.